src/Microsoft.DotNet.CodeAnalysis/PackageFiles/codeAnalysis.targets

```diff
@@ -9,6 +9,8 @@
     <CodeAnalyzerDir Condition="'$(CodeAnalyzerDir)'=='' and '$(OSEnvironment)'!='Windows_NT'">$(ToolsDir)analyzers/</CodeAnalyzerDir>
   </PropertyGroup>
   
+  <Import Condition="'$(EnableDotnetAnalyzers)' == 'true'" Project="$(MSBuildThisFileDirectory)PinvokeAnalyzer.props"/>
```

Comment:
As Alex suggested above, instead of having this import we should just move the contents of your props file here instead.